framework/include/wifi_manager/wifi_manager.h

```diff
@@ -228,8 +228,35 @@ wifi_manager_result_e wifi_manager_disconnect_ap(void);
  */
 wifi_manager_result_e wifi_manager_scan_ap(void);
 
-#endif
+/**
+ * @brief Save the AP configuration at persistent storage
+ * @details @b #include <wifi_manager/wifi_manager.h>
+ * @param[in] AP configuration infomation should be given including ssid, channel, and passphrase.
```

Comment:
When you use param[xx], you should write a name of argument first. That is a rule of doxygen.
```
* @param[in] config AP ......
```

Reply:
Can you give me a example? I just follow the above templete